packages/checksum/index.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+module.exports = require('./checksum');
```

Comment:
I assume the reason for doing this is that this package could expand in the future and it's clearer to organize functionality into well-named files?

Reply:
Actually I organized it this way mainly to retain parity between the test files and the files being tested. `checksum.js` and `test-checksum.js`. As a personal preference I also like keeping `index.js` as an interface-only to document all the exported functionality. But yeah, what you said works too! 😃